includes/admin.form.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @file
+ * Adminstration settings for the Islandora MARCXML module.
+ */
+
+/**
+ * Adminstration form for the Islandora MARCXML module.
+ */
+function islandora_marcxml_admin_settings(array $form, array &$form_state) {
+  $form['islandora_marcxml_allow_editing_of_existing_mods'] = array(
```

Comment:
Whoops... just noticed a thing. Missing the `hook_uninstall()` deleting the new var.